src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -444,6 +444,12 @@
         - name: Create a Windows task from a public GitHub repository which builds the Azure Container Builder image on Amd64 architecture.
           text: >
             az acr task create -t acb:{{.Run.ID}} -n acb-win -r MyRegistry -c https://github.com/Azure/acr-builder.git -f Windows.Dockerfile --commit-trigger-enabled false --pull-request-trigger-enabled false --platform Windows/amd64
+        - name: Create a task from a public GitHub repository which builds and pushes the image in the source registry (i.e. MyRegistry1) as per simpletask.yaml file in the repo.
+          text: >
+            az acr task create -n hello-world -r MyRegistry1 -c https://github.com/shahzzzam/acr-build-helloworld-node.git -f simpletask.yaml --commit-trigger-enabled false --pull-request-trigger-enabled false
```

Comment:
I think we should avoid using personal github repo. Check if you can move the examples to https://github.com/Azure-Samples/acr-tasks?